### PR TITLE
Ed/new route for api interaction testing

### PR DESCRIPTION
**Pull request by edelans**
Add a new route `/seedAccount` to demo API interaction in travelSpot with a realistic use-case (= close to the real life needs of our customers). 

Additionally, I added prettier to the repo, sorry should probably have done this separately for a simpler PR.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,63 +1,112 @@
...
+
+// This route is meant to demonstrate the API interaction in the TravelSpot demo
+router.post("/seedAccount", async (ctx) => {
+  const body = ctx.request.body;
+
+  const email =
+    body && body.email
+      ? body.email
+      : ["test-", randomString(8), "@waldo.com"].join("");
+  const password = body.password ? body.password : randomString(8);
```
Comment (JeremyFuhrmann) [inline index.js:31]: you can check the `body` before checking its `password` field

File: index.js [hunk 2]
```diff
@@ -1,63 +1,112 @@
...
+import bodyParser from "koa-bodyparser";
+import axios from "axios";
 
 const app = new koa();
+app.use(bodyParser());
 const router = new Router();
 
 const CWD = process.cwd();
 
+function randomString(strLength, charSet) {
```
Comment (JeremyFuhrmann) [inline index.js:12]: you could put this function in a separate ùtils` file, or even use an external lib like [here](https://github.com/waldoapp/backend/blob/3f9fcb9a7aa34f74d47148be4164f206644e4b92/packages/backend-utils/src/crypto.js#L26), but not sure it worth it

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,11 @@
+node_modules/
+.idea
```
Comment (JeremyFuhrmann) [inline .prettierignore:2]: what is this file?

**Review by JeremyFuhrmann — APPROVED** [review]
LGTM!
you can also update the readme with this new endpoint

File: .prettierignore (hunk 3 above)
Comment (edelans) [reply .prettierignore:2]: .idea ? I guess it's a config file for an IDE. I just took what pre-existed in the .gitignore, as recommended in [prettier doc](https://prettier.io/docs/en/install.html)

File: index.js [hunk 4]
```diff
@@ -1,63 +1,112 @@
...
+}
+
+// This route is meant to demonstrate the API interaction in the TravelSpot demo
+router.post("/seedAccount", async (ctx) => {
+  const body = ctx.request.body;
+
+  const email =
+    body && body.email
+      ? body.email
+      : ["test-", randomString(8), "@waldo.com"].join("");
```
[the remark this answers is not in the document]
Comment (edelans) [reply index.js:30]: nice, thanks !

File: index.js (hunk 1 above)
Comment (edelans) [reply index.js:31]: good point 👍

File: index.js (hunk 2 above)
Comment (edelans) [reply index.js]: done

**Pull request merged**